src/api/signals/revision.py

```diff
@@ -48,14 +47,8 @@ def update_profile_revisions(sender, instance, *args, **kwargs):
     _create_project_profile_revisions({"profile": instance})
 
 
-@receiver(post_save, sender=ProjectProfile)
-def update_project_profile_revisions(sender, instance, *args, **kwargs):
-    Revision.create_from_instance(instance.project)
-
-
 @receiver(pre_delete, sender=ProjectProfile)
 def delete_project_profile_revisions(sender, instance, *args, **kwargs):
-    Revision.create_from_instance(instance.project)
```

Comment:
OK so we can drop all the project revision creation steps because of the db-level trigger, right? I think it's worth making a comment at the top of this file to remind our future selves about the triggers; I had to spend a few minutes remembering how we'd set this up.